app/views/home/_users.html.erb

```diff
@@ -15,7 +15,11 @@
         <%= user.display_name %>
       </div>
       <div class="user-meeting d-flex">
-        <span><i class="fa fa-phone fa-flip-horizontal"></i></span>
+        <%= form_with model: JitsiCall.new, id: "init-call-#{user.id}" do |f| %>
+          <%= f.hidden_field :user_id, value: user.id %>
+          <%= button_tag(type: 'submit', class: "fa fa-phone fa-flip-horizontal phone-button") do %>
```

Comment:
Have you tried `f.submit` here? Maybe one can omit the `type` then

Reply:
Yes this results in an awful button